roster2/tool/src/java/org/sakaiproject/roster/tool/entityprovider/RosterPOIEntityProvider.java

```diff
@@ -646,20 +646,20 @@ private List<String> createColumnHeader(final String viewType, final String site
 			header.add(rl.getString("facet_email"));
 		}
 
-		if (this.sakaiProxy.getViewCandidateDetails(siteId)) {
-			header.add(rl.getString("facet_studentNumber"));
-		}
-
 		if (this.sakaiProxy.getViewUserProperty(siteId)) {
 			header.add(rl.getString("facet_userProperties"));
 		}
 
 		if (this.sakaiProxy.getViewCandidateDetails(siteId)) {
-			header.add(rl.getString("facet_specialNeeds"));
+			header.add(rl.getString("facet_additionalNotes"));
 		}
 
 		if (this.sakaiProxy.getViewCandidateDetails(siteId)) {
-			header.add(rl.getString("facet_additionalNotes"));
+			header.add(rl.getString("facet_studentNumber"));
+		}
+
+		if (this.sakaiProxy.getViewCandidateDetails(siteId)) {
```

Comment:
Couldn't all these 3 conditions go under the same if?

Reply:
Yes!